### DISCUSSION TITLE
project management datasets

### DISCUSSION THREAD

**Discussion by Adeleke-Obisanya** [catalyst-cooperative/pudl#3882] [Help Me!]
where can i  find a project management datasets for my dissertation on the schedule, budget overrun?

**Comment by zaneselvans** [comment]
This doesn't seem related to PUDL or our other energy data work so I'm going to close.

**Comment by jameswar54502** [comment]
The availability of [project management](https://desktrack.timentask.com/blog/project-management-software/
) datasets is incredibly valuable for both researchers and practitioners! These datasets can provide insights into project performance, resource allocation, and risk factors, helping teams make data-driven decisions. Analyzing historical project data can also reveal trends and best practices, ultimately improving future project outcomes. I’d love to see discussions around specific datasets that have proven useful, as well as techniques for effectively analyzing and visualizing this data. Sharing experiences and findings can greatly benefit the project management community and enhance our understanding of successful project execution.
